site/content/events/2016-losangeles-1day/_email_organizers.txt

```diff
@@ -1 +1 @@
-organizers-losangeles-2016@devopsdays.org
+devopsdayla-planning@googlegroups.com
```

Comment:
My understanding is that the devopsdays.org MLs are preferred; maybe @KrisBuytaert or @bridgetkromhout can confirm the policy (if any) here.